opennms-services/src/main/java/org/opennms/netmgt/collectd/AliasedResource.java

```diff
@@ -207,7 +207,7 @@ public String getInstance() {
 
     @Override
     public ResourcePath getParent() {
-        return null; //For node and interface type resources, use the default parent
+        return getCollectionAgent().getStorageResourcePath();
```

Comment:
Is this change necessary? If so I'd need to take a closer look and see if it has any other impacts.

Reply:
Yeah, this is needed to parse nodeId  for AliasedResource.  
IfInfo does the same way https://github.com/OpenNMS/opennms/blob/develop/opennms-services/src/main/java/org/opennms/netmgt/collectd/IfInfo.java#L273

Reply:
Sounds good.